home_work_3/models.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy import Column, Integer, String, Boolean, ForeignKey, Numeric
+from sqlalchemy.orm import as_declarative, declared_attr
+
+
+@as_declarative()
+class AbstractModel:
+    id = Column(Integer, primary_key=True, autoincrement=True)
```

Comment:
К тому же, раз уж вы создаёте атрибуты через `@declared_attr`, то создавайте их все подобным образом
```suggestion
    @declared_attr
    def id(cls):
        return Column(Integer, primary_key=True)
```